lib/rules/no-global-jquery.js

```diff
@@ -3,6 +3,10 @@
  */
 'use strict';
 
+const { getEmberImportBinding } = require('../utils/imports');
+const { collectObjectPatternBindings } = require('../utils/destructed-binding');
+
+const MESSAGE = 'Do not use global `$` or `jQuery`.';
```

Comment:
Sorry for being late to review, but shouldn't we have kept the guide link in this message?